index.html

```diff
@@ -361,5 +361,27 @@ <h2>Get started</h2>
                 </footer>
             </section>
         </div>
+
+        <div class="wrapper-menu" id="wrapper-menu">
+            <header class="header">
+                <nav class="nav">
+                    <a class="close-menu" href="#"><img class="menu-button" src="img/Union.svg" alt="Menu Buttom"></a>
+                </nav>
+            </header>
+
+            <section class="menu">
+                <div class="content-menu">
+                    <ul class="menu-list">
+                        <li><a class="close-menu-headline" href="#headline"><h4>Hello</h4></a></li>
+                        <li><a class="close-menu-work" href="#works"><h4>Portfolio</h4></a></li>
+                        <li><a class="close-menu-about" href="#about-myself"><h4>About</h4></a></li>
+                        <li><a class="close-menu-contact" href="#contact"><h4>Contact</h4></a></li>
+                    </ul>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_7](https://user-images.githubusercontent.com/69638013/161549897-79ae4299-7803-491c-b90d-63ba91fc5dea.png) | ![Screenshot_6](https://user-images.githubusercontent.com/69638013/161549474-66ae7a8d-6c1f-47e6-a157-a789880e3243.png)


- [ ] Kindly reduce spacing  before and after the nav links, so it looks like the Figma design